handlers/viber/keyboard.go

```diff
@@ -0,0 +1,113 @@
+package viber
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf8"
+)
+
+// KeyboardButton is button on a keyboard, see https://developers.viber.com/docs/tools/keyboards/#buttons-parameters
+type KeyboardButton struct {
+	ActionType string `json:"ActionType"`
+	ActionBody string `json:"ActionBody"`
+	Text       string `json:"Text"`
+	TextSize   string `json:"TextSize"`
+	Columns    string `json:"Columns,omitempty"`
+	BgColor    string `json:"BgColor,omitempty"`
+}
+
+// Keyboard models a keyboard, see https://developers.viber.com/docs/tools/keyboards/#general-keyboard-parameters
+type Keyboard struct {
+	Type          string           `json:"Type"`
+	DefaultHeight bool             `json:"DefaultHeight"`
+	Buttons       []KeyboardButton `json:"Buttons"`
+}
+
+const (
+	// maxColumns refer to the maximum number of columns units in a row
+	maxColumns = 6
+	// maxRowsRunes refer to maximum number of runes in the same row to help define buttons layout
+	maxRowRunes = 38
+	// paddingRunes to help to calculate the size of button column width
+	paddingRunes = 2
+)
+
+// NewKeyboardFromReplies create a keyboard from the given quick replies
+func NewKeyboardFromReplies(replies []string, buttonConfig map[string]interface{}) *Keyboard {
+	rows := StringsToRows(replies, maxColumns, maxRowRunes, paddingRunes)
+	buttons := []KeyboardButton{}
+
+	for i := range rows {
+		for j := range rows[i] {
+			cols := 6 / len(rows[i])
+
+			button := KeyboardButton{
+				ActionType: "reply",
+				TextSize:   "regular",
+				ActionBody: rows[i][j],
+				Text:       rows[i][j],
+				Columns:    fmt.Sprint(cols),
+			}
+
+			button.ApplyConfig(buttonConfig)
+			buttons = append(buttons, button)
+		}
+	}
+
+	return &Keyboard{"keyboard", false, buttons}
+}
+
+//ApplyConfig apply the configs from the channel to KeyboardButton
+func (b *KeyboardButton) ApplyConfig(buttonConfig map[string]interface{}) {
+	bgColor := strings.TrimSpace(fmt.Sprint(buttonConfig["bg_color"]))
+	textStyle := strings.TrimSpace(fmt.Sprint(buttonConfig["text"]))
+	textSize := strings.TrimSpace(fmt.Sprint(buttonConfig["text_size"]))
+
+	if len(bgColor) == 7 {
+		b.BgColor = bgColor
+	}
+	if strings.Contains(textStyle, "*") {
+		b.Text = strings.Replace(textStyle, "*", b.Text, 1)
```

Comment:
Does this mean we need to escape HTML chars here? I'm guessing this breaks for quick replies that contain `<` or `>`

Reply:
I think `html.ScapeString` can prevent problems with HTML chars, so i put on line 51. It's fine, that way?
https://github.com/nyaruka/courier/pull/385/files#diff-f16d69b49e81106d40a4c7ff1068ddd7b399e1b38b29c48c0d3481897479679bR51